aip/0192.md

```diff
@@ -40,10 +40,10 @@ first sentence of each comment **should** omit the subject and be in the
 third-person present tense:
 
 ```proto
-// Creates a book on the given shelf.
+// Creates a book from the given publisher.
```

Comment:
Maybe "under" the given publisher? The "from" here sounds quite odd.

Reply:
```suggestion
// Creates a book under the given publisher.
```